baggage/baggage.go

```diff
@@ -263,7 +263,15 @@ func NewMember(key, value string, props ...Property) (Member, error) {
 	if err := m.validate(); err != nil {
```

Comment:
The function documentation needs to be updated to clearly inform developers what this function expects.

Reply:
The doc suggests function comment? If so, function comment is added for the input value encoding and decoding clarification.